google/cloud/google_cloud_cpp_common_version.bzl

```diff
@@ -18,5 +18,5 @@
 
 GOOGLE_CLOUD_CPP_IS_RELEASE = ""
 GOOGLE_CLOUD_CPP_VERSION_MAJOR = "0"
-GOOGLE_CLOUD_CPP_VERSION_MINOR = "8"
+GOOGLE_CLOUD_CPP_VERSION_MINOR = "11"
```

Comment:
You probably did not run CMake after changing this number from 11 to 9.

Reply:
re-ran cmake and committed the new version